app/services/exports/export_service.rb

```diff
@@ -20,16 +21,19 @@ def export_xml(full_update: false, collection: nil, year: nil)
           users_archives_for_manifest = get_user_archives(start_time, full_update)
         when "organisations"
           organisations_archives_for_manifest = get_organisation_archives(start_time, full_update)
-        else
+        when "lettings"
           lettings_archives_for_manifest = get_lettings_archives(start_time, full_update, year)
+        when "sales"
+          sales_archives_for_manifest = get_sales_archives(start_time, full_update, year)
         end
       else
         users_archives_for_manifest = get_user_archives(start_time, full_update)
         organisations_archives_for_manifest = get_organisation_archives(start_time, full_update)
         lettings_archives_for_manifest = get_lettings_archives(start_time, full_update, year)
+        sales_archives_for_manifest = get_sales_archives(start_time, full_update, year) if Time.zone.now >= Time.zone.local(2025, 4, 1)
       end
 
-      write_master_manifest(daily_run_number, lettings_archives_for_manifest.merge(users_archives_for_manifest).merge(organisations_archives_for_manifest))
+      write_master_manifest(daily_run_number, lettings_archives_for_manifest.merge(sales_archives_for_manifest || {}).merge(users_archives_for_manifest).merge(organisations_archives_for_manifest))
```

Comment:
Will `sales_archives_for_manifest` not be `{}` by default anyway?

Reply:
Ah, yep, good point